docs/templates/.frogbot/frogbot-config.yml

```diff
@@ -12,6 +12,18 @@
       # List of branches to scan
       branches:
         - master
+      # [Optional] Custom formats for commits, branches, and PR created by Frogbot.
+      customFormats:
+        # Please note, Frogbot will replace the following placeholders
+        # 1. IMPACTED_PACKAGE
+        # 2. FIX_VERSION
+        # Example branchName:"(MyLabel):Vulnerability_fix_IMPACTED_PACKAGE_to_version_FIX_VERSION"
+
+        commitTitle: ""
+
+        pullRequestTitle: ""
+
+        branchName: ""
```

Comment:
I suggest changing the template to be - 
```
- params:
    # Git parameters
    git:
      # [Mandatory]
      # Name of the git repository to scan
      repoName: repo-name

      # [Mandatory]
      # List of branches to scan
      branches:
        - master

      # [Optional] Template for the branch name generated by Frogbot when creating pull requests with fixes. The template must include ${BRANCH_NAME_HASH}, to ensure that the generated branch name is unique
      # branchNameTemplate: ""

      # [Optional] Template for the commit message generated by Frogbot when creating pull requests with fixes
      # commitMessageTemplate: ""

      # [Optional] Template for the pull request title generated by Frogbot when creating pull requests with fixes. The template can optionally include the following variables:
      # ${IMPACTED_PACKAGE}
      # ${FIX_VERSION} 
      # pullRequestTitleTemplate: ""
```
Let's add our default values for each variable instead of the existing empty string.